mysqld_exporter.go

```diff
@@ -262,6 +264,13 @@ const (
 		  FROM information_schema.schemata
 		  WHERE SCHEMA_NAME NOT IN ('mysql', 'performance_schema', 'information_schema')
 		`
+	queryResponseCheckQuery = `SELECT @@query_response_time_stats`
+	queryResponseTimeQuery = `
+		SELECT
+		    TIME, COUNT, TOTAL
+		  FROM information_schema.query_response_time
```

Comment:
It looks like a histogram will not work in this case.

I need 2 histograms: one to expose COUNT column (number of queries per length) and one to expose TOTAL one (total time of queries per length).
For histogram the buckets should be typed as `map[float64]uint64`
This works fine with COUNT but not with TOTAL as I can't map TIME (float) -> TOTAL (uint) because I will loose the precision unless I multliply TOTAL by 10^6 and keep than in mind when working with those metrics.

What do you think?